micro-s3/src/main/java/com/aol/micro/server/s3/S3Utils.java

```diff
@@ -0,0 +1,40 @@
+package com.aol.micro.server.s3;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import com.amazonaws.services.s3.AmazonS3Client;
+import com.amazonaws.services.s3.model.ListObjectsRequest;
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+
+public class S3Utils {
```

Comment:
We currently use S3ClientProvider to get instance of s3Client (it is possible that someone will want multiple s3 clients for multiple credentials), so we either need to change API to use just one client in plugin or make new client for every S3Utils (in that case all instances will work with one chosen aws credentials).